tests/sql/group_by.slt

```diff
@@ -21,26 +21,35 @@ select v2 + 1, sum(v1) from t group by v2 + 1
 2 3
 3 7
 
-query II rowsort
+query III rowsort
 select sum(v1), v2 + 1 as a, count(*) from t group by a
 ----
 3 2 2
 7 3 2
 5 4 1
 
-query II rowsort
+query III rowsort
```

Comment:
We need to verify if provided column number is the same as user input? cc @wangrunji0408

Reply:
I believe it should verify. But now the database returns a string, which makes it hard to split columns (